ros_buildfarm/templates/ci/ci_job.xml.em

```diff
@@ -420,8 +420,16 @@ parameters = [
     'archive_artifacts',
     artifacts=[
       'ros%d-%s-linux-%s-%s-ci.tar.bz2' % (ros_version, rosdistro_name, os_code_name, arch),
-    ],
+    ] + show_results_images,
+))@
+@[if show_results_images]@
+@(SNIPPET(
+    'image_gallery',
+    title='Performance Test Results',
```

Comment:
Not sure if we should just use a generic title here or if it makes sense to make the configuration more complicated and assign a specific custom title to potentially different sets of image file paths?